core/settings.py

```diff
@@ -57,8 +58,12 @@
 SIMPLE_JWT = {
     'ACCESS_TOKEN_LIFETIME': timedelta(minutes=30),
     'REFRESH_TOKEN_LIFETIME': timedelta(days=1),
+    'BLACKLIST_AFTER_ROTATION': True,
+    'ROTATE_REFRESH_TOKENS': True,
     'AUTH_HEADER_TYPES': ('Bearer',),
-    'AUTH_TOKEN_CLASSES': ('rest_framework_simplejwt.tokens.AccessToken',),
+    'AUTH_TOKEN_CLASSES': 
+    ('rest_framework_simplejwt.tokens.AccessToken'
+    'rest_framework_simplejwt.tokens.RefreshToken',),
```

Comment:
There appears to be a formatting issue in the 'AUTH_TOKEN_CLASSES' setting. The tuple is missing a comma and a line break between the two token classes. It should look like this:
```
'AUTH_TOKEN_CLASSES': (
    'rest_framework_simplejwt.tokens.AccessToken',
    'rest_framework_simplejwt.tokens.RefreshToken',
),
```
Please correct this to avoid runtime errors.